naming/src/main/java/com/alibaba/nacos/naming/constants/Constants.java

```diff
@@ -76,9 +76,9 @@ public final class Constants {
     public static final long ACK_TIMEOUT_NANOS = TimeUnit.SECONDS.toNanos(10L);
     
     /**
-     * The custom instance id key.
+     * The instance id key.
      */
-    public static final String CUSTOM_INSTANCE_ID = "customInstanceId";
+    public static final String INSTANCE_ID = "instanceId";
```

Comment:
I'm not sure that is this a breaking change. If we change the `public static` variable, the previous user upgrade to the new version, are there some differeces?